### ISSUE TITLE
I think Guake is one of the best Quake mode terminals out there

### ISSUE THREAD

**Issue by vadasambar** [Guake/guake#1696]
I did not see a slack channel or Contact Us page anywhere so thought I would create an issue for this. 
I think Guake is one of the best Quake mode terminals out there. It is super responsive, supports multiple windows and has really easy to remember keyboard shortcuts for creating and switching tabs. 

I really want to thank you guys for creating this slick piece of software :D

**Comment by hackel** [comment]
> I did not see a slack channel or Contact Us page anywhere so thought I would create an issue for this.

Don't do this.  You are literally just creating more work for the project maintainers.  I understand your good intentions, but there are over 400 open issues in this repo.  Please close this issue at once.

**Comment by vadasambar** [comment]
@hackel I have closed the issue.
>Don't do this. You are literally just creating more work for the project maintainers.
I don't understand how this is more work, except reading and closing this issue. I understand that reading and closing an issue can be work but I created an issue because I had no other means to communicate/appreciate the work that is being done here. 

If you think I should have communicated this some other way, I would like to know more about that. 

Also, you are literally telling me to not appreciate the software I use everyday without giving me any other means of communication. I am very sorry but `Don't do that` makes me feel like I should not post anything good about Guake here. It feels discouraging.
